.github/renovate.json

```
@@ -16,6 +16,7 @@
   ],
   "packageRules": [
      {
+       "matchPackagePatterns": ["*"],
```

Looks like it wants a [regex](https://docs.renovatebot.com/configuration-options/#matchpackagepatterns)
```suggestion
       "matchPackagePatterns": [".*"],
```